docs/articles/getting-started.md

```diff
@@ -44,12 +44,13 @@ private async void button1_Click(object sender, EventArgs e)
 }
 ```
 
-If you then run your application and click the button you will see that all your chroma devices turned Red.
+If you then run your application and click the button you will see that all your chroma devices turned red.
+What `await` does is waiting for the initializiation of the Chroma SDK to finish before proceding. Methods using `await` not in the last line must be marked ss `async` (asynchronous) and thus allow the usage of `await` on the method itself. How this works under the hood, is that [Task](https://docs.microsoft.com/dotnet/csharp/programming-guide/concepts/async/) objects are returned instead of return values.
 
-### A note on [`ColoreProvider`](xref:Colore.ColoreProvider)
+#### A note on `ColoreProvider`
 
 Since version 6.0.0, Colore is designed to be asynchronous and less strict in how it is used.
-This means that the [`Chroma`](xref:Colore.IChroma) instance is now obtained differently.
+This means that the `Chroma` instance is now obtained differently.
```

Comment:
Why was the `xref` removed here and above for `ColoreProvider`?

Reply:
I just added my 5 cents to the wiki and copied it into the docs folder.